examples/langserver_ext.py

```diff
@@ -7,6 +6,11 @@
 
 from pyls_jsonrpc import streams
 
+try:
+    import ujson as json
+except ImportError:
```

Comment:
Please change this `ImportError` to catch all possible errors generated when importing `ujson`. The thing is more complex errors can be thrown for compiled dependencies.